docs/maintenance.txt

```diff
@@ -55,3 +55,9 @@ That's it. In the execute() method, you have access to all of the normal
 MediaWiki functions, so you can get a DB connection, use the cache, etc.
 For full docs on the Maintenance class, see the auto-generated docs at
 http://svn.wikimedia.org/doc/classMaintenance.html
+
+Note:
```

Comment:
I'd document an example implementation here. We may retire FounderEmails at some point or refactor it which will destroy the example. So it's good to have it here. It'll also be simpler to grasp by a person willing to create a cronjob.